app.rb

```diff
@@ -17,17 +17,6 @@ def initialize
     @rentals = []
   end
 
-  def menu
-    puts
-    45.times { print '=' }
-    puts "\nPlease choose an option by entering a number:"
-    45.times { print '-' }
-    puts "\n"
-    print "1. List all books\n2. List all people\n3. Create a person\n" \
-          "4. Create a book\n5. Create a rental\n6. List all rentals for" \
-          "a given person id\n7. Exit\n"
-  end
-
   def create_book
     print 'Title: '
     title = gets.chomp
```

Comment:
- [ ] In this method you are getting input from users and creating the book, This method does not follow `single responsibility principle`. Please make sure to separate these operations into different methods to achieve `single responsibility principle`.